vscode/src/test/suite/general/extension.test.ts

```diff
@@ -197,7 +197,7 @@ suite('Extension Test Suite', function () {
       assert.strictEqual(tests[1].tests[0].name, 'testTrue', `Invalid test name returned`);
 
       console.log("Test: run all workspace tests");
-      await vscode.commands.executeCommand(myExtension.COMMAND_PREFIX + '.run.test', workspaceFolder.uri.toString());
+      await vscode.commands.executeCommand(extConstants.COMMAND_PREFIX + '.run.test', workspaceFolder.uri.toString());
```

Comment:
Should we replace the command names with the references from *commands.ts*?

Reply:
Done